app/components/Notifier/Notifier.jsx

```diff
@@ -36,14 +33,6 @@ export default class Notifier extends Component
         type        : 'alert'
     }
 
-    handleDismiss = () =>
```

Comment:
This doesn't seem right. We're removing a general 'ignore' function for the browser due to safe specific issues...

I'd say we still need this func to be available. And if we want something else to happen on ignore, we add that functionality it.